.gitignore

```diff
@@ -5,6 +5,7 @@ bin/
 
 .project
 .plugins/.project
-
+plugins/org.jboss.tools.openshift.client/.project
```

Comment:
why would you want to ignore those Eclipse configuration files (all of them, the ones below, too)?